Description: Remove embeded C libraries in bundled crates
Author: Zixing Liu <zixing.liu@canonical.com>
Forwarded: not-needed
Last-Update: 2023-05-17
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: rustc/src/tools/cargo/Cargo.toml
===================================================================
--- rustc.orig/src/tools/cargo/Cargo.toml
+++ rustc/src/tools/cargo/Cargo.toml
@@ -206,8 +206,6 @@ test = false
 doc = false
 
 [features]
-vendored-openssl = ["openssl/vendored"]
-vendored-libgit2 = ["libgit2-sys/vendored"]
+# Debian: removed vendoring flags
 pretty-env-logger = ["pretty_env_logger"]
 # This is primarily used by rust-lang/rust distributing cargo the executable.
-all-static = ['vendored-openssl', 'curl/static-curl', 'curl/force-system-lib-on-osx']
